OpenGL/Constructs/VBO.cs

```diff
@@ -134,7 +132,7 @@ private set
         /// <summary>
         /// Specifies whether the VBO contains an integral type.
         /// </summary>
-        public bool IsIntegralType { get; }
```

Comment:
I think these should be marked as protected still.  We shouldn't be allowed user code to change this.